hoomd/md/pair/aniso.py

```diff
@@ -328,6 +328,54 @@ class ALJ(AnisotropicPair):
     only ``vertices`` creates a convex polytope (set ``vertices`` and ``faces``
     to empty lists to create the ellipsoid).
 
+    Important:
+        `ALJ` implicitly rounds the given shape by :math:`\sigma_c` via the WCA
+        or Lennard-Jones potential.
+
+    .. rubric:: Choosing `r_cut <hoomd.md.pair.Pair.r_cut>`:
+
+    Set `r_cut <hoomd.md.pair.Pair.r_cut>` for each pair of particle types so
+    that `ALJ` can compute interactions for all possible relative placements
+    and orientations of the particles. The furthest apart two particles can be
+    while still interacting depends on the value of ``alpha`` (where :math:`R_i`
+    is the circumsphere radius of the particle with type :math:`i`):
```

Comment:
Replace circumsphere with "distance from the origin to the furthest point". This is only the circumsphere radius when the particle's origin is the center of the circumsphere.

Reply:
What if we used coxeter language and described R_i as the "radius of the shape's minimal centered bounding sphere"?